frontend/components/TokenTransfers.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div class="token-transfers list-view">
+    <div v-if="loading" class="text-center py-4">
+      <Loading />
+    </div>
+    <div v-else-if="transfers.length === 0" class="text-center py-4">
+      <h5>No transactions found</h5>
+    </div>
+    <div v-else>
+      <Table>
+        <THead>
+          <Cell>Transaction</Cell>
+          <Cell>Extrinsic</Cell>
+          <Cell>From address</Cell>
+          <Cell>To address</Cell>
+          <Cell>Amount</Cell>
+          <Cell>Timestamp</Cell>
+          <Cell align="center">Success</Cell>
+        </THead>
+
+        <Row v-for="(item, index) in list" :key="index">
+          <Cell :link="`/contract/tx/${item.extrinsic.hash}`">{{
+            shortHash(item.extrinsic.hash)
+          }}</Cell>
+
+          <Cell
+            :link="`/extrinsic/${item.extrinsic.block_id}/${item.extrinsic.index}`"
+          >
+            #{{ formatNumber(item.extrinsic.block_id) }}-{{
+              formatNumber(item.extrinsic.index)
+            }}
+          </Cell>
+
+          <Cell
+            :link="{ url: `/account/${item.from_address}`, fill: false }"
+            :title="$t('pages.accounts.account_details')"
+          >
+            <ReefIdenticon
+              :key="item.extrinsic.signer"
+              :address="item.from_address"
+              :size="20"
+            />
+            <span>{{ shortAddress(item.from_address) }}</span>
+          </Cell>
+
+          <Cell
+            :link="{ url: `/account/${item.to_address}`, fill: false }"
+            :title="$t('pages.accounts.account_details')"
+          >
+            <ReefIdenticon
+              :key="item.extrinsic.signer"
+              :address="item.to_address"
+              :size="20"
+            />
+            <span>{{ shortAddress(item.to_address) }}</span>
+          </Cell>
+
+          <Cell>{{ formatAmount(item.amount) }}</Cell>
```

Comment:
All transfer formatAmount calls need to pass symbol and decimals to display the right  symbol.
This values are obtained by adding `token {
                address
                verified_contract {
                  contract_data
                }
              `} to transfer GQL - like in AccountTransfers.

Reply:
Can fix in new PR?